extra/1-currency-conversion.js

```diff
@@ -15,7 +17,11 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(num) {
+  const feeDeductedAmount = num - (num * 0.01);
+  const brlValue = Number(feeDeductedAmount * (5.7)).toFixed(2);
```

Comment:
nice use of `toFixed()`. You don't need the `Number` object here as it's already a number at that point, as the `toFixed` function turns the value into a string. So this can be simplified to
```
const brlValue = (feeDeductedAmount * 5.7).toFixed(2);
```